keyword_checker.js

```diff
@@ -0,0 +1,68 @@
+module.exports = {
+    /*
+    * Scans input string for convention violations and returns feedback based on
+    * detected violations
+    * Returns a string containing the formatted feedback message
+    */
+    getFeedback: function (string) {
+        message = "";
+        
+        if (detectSpaceBetweenHashtagAndDigit(string)) {
+            console.log("Detected space between # and digit");
+            message += getFormattedMessage(getMessageSpaceBetweenHashtagAndDigit());
+        } else if (!detectIssueReference(string)) {
+            console.log("Issue reference not found");
+            message += getFormattedMessage(getMessageIssueReferenceMissing());
+        }
+        
+        if (!detectGithubKeyword(string)) {
+            console.log("Missing GitHub keyword");
+            message += getFormattedMessage(getMessageGithubKeywordMissing());
+        }
+        
+        return message;
+    }
+};
+
+/*
+* Checks if an issue reference is present in argument
+* Returns true if present, false otherwise
+*/
+function detectIssueReference(string) {
+    referenceTest = new RegExp(/[\s\S]*#\d[\s\S]*/);
+    return referenceTest.test(string);
+}
+
+/*
+* Checks if argument contains a space between a # and a digit
+* Returns true if present, false otherwise
+*/
+function detectSpaceBetweenHashtagAndDigit(string) {
+    spaceTest = new RegExp(/[\s\S]*# \d[\s\S]*/);
+    return spaceTest.test(string);
+}
+
+/*
+* Detects if a GitHub keyword is present in argument
+* Returns true if present, false otherwise
```

Comment:
Can be shortened to `Returns true if the argument contains one of GitHub keywords ...`
Also, specify the keywords.